ospfd/ospf_network.c

```diff
@@ -216,12 +193,6 @@ int ospf_sock_init(struct ospf *ospf)
 		exit(1);
 	}
 
-	ret = ospf_bind_vrfdevice(ospf, ospf_sock);
```

Comment:
shouldn't we be calling vrf_bind?  Why are we loosing this  call?

Reply:
code moved to lib/vrf.c, behind vrf_socket call()